pkgs/data/fonts/azeret-mono/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib, stdenvNoCC, fetchFromGitHub }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "azeret-mono";
+  version = "1.0";
+
+  src = fetchFromGitHub {
+    owner = "displaay";
+    repo = "Azeret";
+    rev = "3d45a6c3e094f08bfc70551b525bd2037cac51ba";
+    sha256 = "WC5a2O+/hdX+lLz81obcmq64wYpX48ZxsYPEaZUbFaY=";
```

Comment:
Prefer `hash = ...` to `sha256 = ...` Both currently work, but the former will make it easier if the algorithm ever needs to change.

Since there could be different hashing algorithms in use, this make sense.

Reply:
great point, will do